package.json

```diff
@@ -16,7 +16,7 @@
     "test": "./node_modules/.bin/mocha --harmony --compilers js:babel-register 'node-tests/**/*-test.js'"
   },
   "engines": {
-    "node": "^4.5 || 6.* || 7.* || 8.*"
+    "node": "^4.5 || >=6.*"
```

Comment:
Is there any reason the engines where so strict before? Could change to `^4.5 || 6.* || 7.* || 8.* || 9.*` or `^4.5 || >=6.* <= 9.*` if you prefer that one but I don't see any reason to prevent user from using it with upcoming node releases until we know that there are bugs.

Reply:
This works as well - I don't recall why we initially made it so strict. Thanks for the PR.